### DISCUSSION TITLE
Train multiple loras sequentially?

### DISCUSSION THREAD

**Discussion by PaperOrb** [bmaltais/kohya_ss#705] [Q&A]
I'm wondering if we can create a few kohya settings jsons and then have kohya load them sequentially so you can have it train multiple loras one after the other overnight?

edit: nm, found out that lora easy training scripts has this functionality.

**Comment by Kadah** [comment]
I'd like to see this functionality here. More so if it could handle a pool of more than one GPU.

A workaround I think would work is use the `Print training command` function and build a basic script to execute them outside of the gui.

**Comment by Romangelo** [comment]
I'd love to have this. I could train 7-8 loras each night instead of just one that I pick before going to bed.
